package.json

```diff
@@ -65,12 +65,17 @@
     "test": "npm run cover"
   },
   "dependencies": {
+    "@google-cloud/automl": "^0.1.1",
     "@google-cloud/promisify": "^0.3.0",
+    "argparse": "^1.0.10",
     "google-gax": "^0.18.0",
     "is": "^3.2.1",
+    "lodash": "^4.17.10",
     "lodash.merge": "^4.6.1",
+    "mathjs": "^5.0.4",
     "protobufjs": "^6.8.6",
-    "yargs": "^12.0.0"
+    "util": "^0.11.0",
```

Comment:
Was this on purpose?

Reply:
yes!  This and mathjs are required by package- but like l can move these to samples/package.json